src/pkg/backup/details/sharepoint.go

```diff
@@ -52,12 +52,9 @@ type SharePointInfo struct {
 }
 
 type ListInfo struct {
-	Name      string    `json:"name,omitempty"`
-	ItemCount int64     `json:"itemCount,omitempty"`
-	Template  string    `json:"template,omitempty"`
-	WebURL    string    `json:"webUrl,omitempty"`
-	Created   time.Time `json:"created,omitempty"`
-	Modified  time.Time `json:"modified,omitempty"`
+	Name      string `json:"name,omitempty"`
```

Comment:
Given this is altering persisted state, we should merge this PR before the PR which re-enables backup for lists.